### DISCUSSION TITLE
Внешняя форма

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#488] [ОШБК!] [answered]
Добрый день. Столкнулся с ошибкой, при созданий внешних форм. 
После созданий формы, и переходе по ссылке нечего не грузится "Не удается открыть эту страницу"

![Безымянный](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/3246c504-d78b-40d0-888a-ca86fd1d8884)

Что было сделано: 
1. Изучил тему на офф.сайте : [](https://ru.docs.totum.online/forms)
2. Изучил посмотрел 1-3 часть видео в ютубе. (+ первую 2 раза пересмотрел.)
3. Нашел тему на форуме, #351  (успешно обновился, очистил кеш браузера)
4. Не помогло.

В чем может быть проблема?

**Comment by ttm-alexeypolunin** [comment]
Недостаточно информации. Те `nginx` отклонил соединение, но что вы там делаете и как установлено — неизвестно, неизвестно.

**Reply by ttm-alexeypolunin** [reply, marked as answer]
Я могу предположить, что ссылка то в интерфейсе генерируется с учетом `https`, а если у вас там по ip установлено, то у `nginx` в этом случае  прописан `80` порт тк `ssl` нет.

**Comment by Re-Requeste** [comment]
> Я могу предположить, что ссылка то в интерфейсе генерируется с учетом `https`, а если у вас там по ip установлено, то у `nginx` в этом случае прописан `80` порт тк `ssl` нет.
Да так и есть. Поменял https на http прошла болячка, благодарю
